pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/sql.tsx

```diff
@@ -40,6 +40,26 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph
+      title="Open SQL Transactions"
+      sources={nodeSources}
+      tooltip={`The total number of SQL transactions currently open ${tooltipSelection}.`}
+    >
+      <Axis label="transactions">
+        <Metric name="cr.node.sql.txns.open" title="Open Transactions" />
+      </Axis>
+    </LineGraph>,
+
+    <LineGraph
+      title="Active SQL Queries"
+      sources={nodeSources}
+      tooltip={`The total number of SQL queries currently running ${tooltipSelection}.`}
+    >
+      <Axis label="queries">
+        <Metric name="cr.node.sql.distsql.queries.active" title="Active Queries" />
```

Comment:
drive-by comment -- does this metric capture all queries (not just distSQL)? is that new or the same?

should the comment on the `sql.distsql.queries.active` metric be updated in execinfra/metrics.go to say this?

Reply:
Yeah im curious on both of these points too

Reply:
All queries, yes